### ISSUE TITLE
oimplib issue? identical result w/wo "-C"

### ISSUE THREAD

**Issue by GitMensch** [LADSoft/OrangeC#643]
In general when using `oimplib some.l some.dll` the generated some.l is not identical when compared with the same generated with the `-C` option.
Using the `-C` option I can create import libraries that work from MSVC generated DLLs. In one single case this is not true - the output files are both identical and linking does not work.

To reproduce:

* download https://ci.appveyor.com/api/projects/GitMensch/gnucobol-3-x-vs/artifacts/dependencies-x86.7z?job=Image:%20Visual%20Studio%202019;%20Configuration:%20Release;%20Platform:%20x86
* use `oimplib -C` on cJSON.dll - it is identical to doing the same without `-C`
* linking the following program does not work:

```c
  #include <cJSON.h>
  #if (CJSON_VERSION_MAJOR * 100 + CJSON_VERSION_MINOR) < 103
     	#error Needs at least cJSON 1.3.0
  #endif
 int
 main ()
 {
   cJSON_CreateNull ();
   return 0;
 }
```

> Error: Undefined External 'cJSON_CreateNull' in module conftest.c

**Comment by LADSoft** [comment]
the problem was microsoft-style exports.   I've fixed that, and deprecated the -C option.   Everything is automatic, you don't need -C, but, if you use -C it will be ignored.

**Comment by LADSoft** [comment]
fixed in this commit : 3bcb1ff484e88272669bae89b5577143502c7701
